manim/mobject/number_line.py

```diff
@@ -19,6 +21,236 @@
 
 
 class NumberLine(Line):
+    def __init__(
```

Comment:
I'd really like to see a basic docstring for the `NumberLine` class; it doesn't have to include examples -- but a `Parameters`-section where each of the available parameters is briefly explained would be great.